experimental/plugins/lando-platformsh/scripts/psh-recreate-users.sh

```diff
@@ -1,3 +1,5 @@
+# This script assures there is a `web` and `app` user
+# and that the `web` user is assigned the uid of 501.
```

Comment:
@mikemilano this all looks great and very much appreciate the comments to help me walk through the logic. Only suggestion is to modify 

> and that the `web` user is assigned the uid of 501.

Since 501 is not universal and probably only true for the first user account on a mac

Reply:
good catch, this has been updated.

Reply:
nice. thank you kind sir!